templates/tfengine/components/terraform/variables/variables.tf

```diff
@@ -1,6 +1,8 @@
+{{range .VARS}}
```

Comment:
add - in {{}}?

Reply:
Adding - actually puts no blank lines so I left this in